src/Dto/Traits/DtoHelper.php

```diff
@@ -12,8 +12,31 @@ protected function setterByData(array $data): void
             }, $k);
             if (method_exists($this, 'set' . $key)) {
                 $this->{'set' . $key}($v);
+            } else {
+                $key = lcfirst($key);
+                $this->$key = $v;
             }
         }
     }
 
+    protected function getterByAttribute(string $attribute)
```

Comment:
Shouldn't this be written as PHP magic __get method?

Reply:
...or maybe not if you want it to be `protected`, because __get and __set must be public

Reply:
The idea was that all of this should be prepared from the resulting table. If there is a need to get data outside, then a dedicated get is created, if not, the idea is not to change the attributes anywhere except the class itself and its extensions